src/test/java/de/tum/in/www1/artemis/connectors/LtiDeepLinkingServiceTest.java

```diff
@@ -0,0 +1,150 @@
+package de.tum.in.www1.artemis.connectors;
```

Comment:
```suggestion
package de.tum.in.www1.artemis.connectors.lti;
```
If the package matches the production package structure methods only used in tests could be made package-private.